data/lang/equipment-core/en.json

```diff
@@ -315,34 +319,66 @@
     "description": "Name for a multiple-missile external missile rack. 'Hydri' is a brand name.",
     "message": "LH-120 Hydri Missile Rack"
   },
+  "MISSILE_RACK_221_DESCRIPTION": {
+    "description": "Equipment description",
+    "message": "External missile rack able to carry up to 2 small missiles."
+  },
   "MISSILE_RACK_322": {
     "description": "Name for a multiple-missile external missile rack. 'Hydri' is a brand name.",
     "message": "LH-230 Hydri Missile Rack"
   },
+  "MISSILE_RACK_322_DESCRIPTION": {
+    "description": "Equipment description",
+    "message": "External missile rack able to carry up to 2 medium-sized missiles. Bigger bangs for your bucks!"
```

Comment:
I'd like the sillyness to be in the sales-pitch. I assume this string is for tooltip?
I'm all for silliness, but we've usually (always?) kept it out from the tooltip.

Reply:
This is the description on the equipment-purchase page. It's also the tooltip when hovering over the equipment on the same page. (I discovered that this is used for a tooltip after seeing it used for the description..)


![image](https://github.com/user-attachments/assets/ba4b3f0c-0522-4ccd-bc21-75a24e9b514e)

Reply:
IMO the tooltip / short description should ideally be a single sentence - at most two, if they're short.

"External missile rack able to carry up to 2 medium-sized missiles." <- Impartial description of what the item is/does; should be used as tooltip / item description.

"Bigger bangs for your bucks!" <- Colorful flavor text, product reviews, information about the manufacturer or item's history. Generally should not be displayed as part of the tooltip.

Reply:
Fixed in #5993 

The "_DESCRIPTION" is used for equipment tooltips and to start off the description blurb in the side-panel now. The "_FLAVOURTEXT", if present, is displayed underneath that and can add anything additional.

Reply:
FWIW, I'm not a huge fan of the tooltips hovering over everything.. I think I'd prefer the hovering to auto-update the right-hand panel instead..

Reply:
@bszlrd thoughts? IMO we'd need to either change the background color or in some way indicate that the stats shown are from the hovered item and not the selected item, or it could get confusing quickly.

Reply:
I wouldn't change anything on the right panel on hover. I think that is not good UX, makes things a bit unpredictable, jumpy. And now you would have to keep the  mouse on the item to see its data.
Do we even need a tooltip in this case?

Reply:
Personally I find the tooltips a bit annoying as they tend to hover over the top of the information on the card which I _do_ want to see.  But I also see the point of the tooltips in being able to quickly browse through a list and get some basic information without having to click on each item.

But at the end of the day, the player will have to click on the item to either purchase it, or to look at the detailed info which isn't in the tooltip anyway.

I've seen quite a few UIs which use the detail-pane-on-hover-over-list-item approach which is why I mentioned it.  If you want to look at an item in more detail, you can just click on it to actually select it and "pin" it to the detail pane - my thinking was that the selected item is the one displayed unless the player hovers over something else.

It looks like it could be implemented pretty easily. But I see the point on it being potentially a "jumpy" and disorienting experience for people not used to this kind of interface.

Reply:
I agree, tooltips hiding the stuff under them is not good either. I never found that detail pane approach you mention good. They feel like an artifact of console UX, where you don't have a proper mouse, so you have to do these odd gymnastics. 
But on the other hand, it could very well work, I just can't imagine it. How complex it would be to prototype it for testing?
Or how about having a hover portion of that screen that shows it, instead of a tooltip?

Another approach I could imagine, but not fully sure about:
Expanding the equipment card downrads to show the description:
![equip-hover-1](https://github.com/user-attachments/assets/fbc1c0fb-e529-4637-b8da-113489b652e1)
![equip-hover-2](https://github.com/user-attachments/assets/7177772b-e154-4fc8-bff5-5c8490dd36bf)
It would still be jumpy in a way, but I not excessively I think. The expansion should only be one line at most, so the height change upon going over a new card wouldn't make it too unpredictable. With keyboard navigation it wouldn't make a difference. And maybe there vould be a very quick animation to it to make it easier on the eyes. I think there are games having a similar UI solution, but I can't recall any on top of my head.
Not sure how complex it would be from the implementation side though.